datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/datacontract/EntityDataContractResolver.java

```diff
@@ -83,7 +83,7 @@ public CompletableFuture<DataContract> get(DataFetchingEnvironment environment)
 
               if (entityResponse != null) {
                 // Step 4: Package and return result
-                return DataContractMapper.mapContract(entityResponse);
+                return DataContractMapper.mapContract(context, entityResponse);
```

Comment:
bringing this mapper in line with other standard mappers and having it accept context